packages/api-folders/src/context/index.ts

```diff
@@ -0,0 +1,14 @@
+import createEntriesContext from "./entries";
+import createFoldersContext from "./folders";
+
+import { Folders, FoldersConfig } from "~/types";
+
+export default async (config: FoldersConfig): Promise<Folders> => {
```

Comment:
Let's use a named export here, `createContext`, because we're exporting a function, and IDE can find usages reliably, and it also helps with refactors. In general, we're trying to avoid default exports/imports on functions/components, and when you see it in the existing code, it was a bad decision made in the past years.